Source/ToolCore/JSBind/JSBTypeScript.cpp

```diff
@@ -57,6 +65,9 @@ String JSBTypeScript::GetScriptType(JSBFunctionType* ftype)
         scriptType = "string[]";
     }
 
+    if (ftype->type_->asHeapPtrType())
+        scriptType = "any";
+
```

Comment:
On second thought, I would suggest that the default for non-specific types should probably be any, to at least allow user TS to compile without changes to engine code. I realize it makes picking up new types that should have specific outputs here a bit tougher, but it's the better approach for user friendliness IMO